paddle/fluid/inference/api/paddle_api.h

```diff
@@ -169,7 +169,8 @@ class PaddlePredictor {
 
   // The common configs for all the predictors.
   struct Config {
-    std::string model_dir;  // path to the model directory.
+    std::string model_dir;    // path to the model directory.
+    void SetNumThreads(int);  // number of threads for each instance
```

Comment:
SetMklNumThreads  or something else? 
The `SetNumThreads` may make user confused that it controls the overall behavior of the predictor, including CPU and GPU.